app/src/main/java/org/mozilla/fenix/home/intent/DeepLinkIntentProcessor.kt

```diff
@@ -33,7 +35,8 @@ class DeepLinkIntentProcessor(
     private val logger = Logger("DeepLinkIntentProcessor")
 
     override fun process(intent: Intent, navController: NavController, out: Intent): Boolean {
-        val scheme = intent.scheme?.equals(BuildConfig.DEEP_LINK_SCHEME, ignoreCase = true) ?: return false
+        val scheme =
```

Comment:
We should avoid modifying the existing if not for needed changes as otherwise the history will be skewed unnecessarily and make blaming changes harder.

Reply:
Reverted this unnecessary code format.